modules/layers/src/path-layer/path-layer-vertex.glsl.js

```diff
@@ -23,12 +23,13 @@ export default `\
 
 attribute vec3 positions;
 
+attribute float instanceTypes;
```

Comment:
This can only take 3 or 4 values right? You could cut the size of this attribute by 3/4 by using uint/UNSIGNED_BYTE instead.